config/studio/content-types/page/post/form-definition.xml

```diff
@@ -426,7 +511,9 @@
             </property>
             <property>
               <name>contentTypes</name>
-              <value>,/component/image,/component/responsive_columns,/component/rich_text,/component/slider</value>
+              <value>
+                ,/component/image,/component/responsive_columns,/component/rich_text,/component/slider
```

Comment:
Remove the `contentTypes` property. This is only for _next_ and it now comes in the form of a datasource